src/algebra/ordered_group.lean

```diff
@@ -700,9 +700,20 @@ le_of_forall_le_of_dense $ λ c hc,
 calc a ≤ b + (c - b) : h _ (sub_pos_of_lt hc)
    ... = c           : add_sub_cancel'_right _ _
 
+lemma le_iff_forall_pos_le_add [densely_ordered α] {x y : α} : x ≤ y ↔ ∀ ε > 0, x ≤ y + ε :=
+⟨λ h ε ε_pos, le_trans h $ le_add_of_nonneg_right $ le_of_lt ε_pos, le_of_forall_pos_le_add⟩
```

Comment:
```suggestion
⟨λ h ε ε_pos, le_add_of_le_of_nonneg h ε_pos.le, le_of_forall_pos_le_add⟩
```
golfed: we have lemmas for `le_trans` combined with `le_add_of_nonneg_right`, might as well use them!